x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_dashboards_view/index.tsx

```diff
@@ -182,6 +190,7 @@ export function StreamDetailDashboardsView({
             <EuiButton
               data-test-subj="streamsAppStreamDetailAddDashboardButton"
               iconType="plusInCircle"
+              disabled={!canLinkAssets}
```

Comment:
@patpscal should a button be disabled or hidden when user does not have necessary permissions ?